modules/nf-core/gzrt/main.nf

```diff
@@ -22,10 +22,10 @@ process GZRT {
         error "GZRT works with .gz files only."
     }
 
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    prefix = task.ext.prefix ?: "${meta.id}_recovered"
+    if ("${fastqgz}" == "${prefix}.fastq.gz") error "Input and output names are the same, set prefix in module configuration to disambiguate!"
```

Comment:
Consider using the common error message: https://github.com/nf-core/modules/blob/8a01490596dbe8443bd6bc6ae4207ae9485f33e3/modules/nf-core/gffread/main.nf#L32

Reply:
corrected, thanks